src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs

```diff
@@ -473,6 +492,8 @@ protected override CompilationOptions CommonWithPlatform(Platform platform)
             return WithPlatform(platform);
         }
 
+        protected override CompilationOptions CommonWithPublicSign(bool publicSign) => WithPublicSign(publicSign);
```

Comment:
I don't have a strong preference between `=>` and `{}` for single line methods.  But it does irk the OCD part of my brain when we have types that mix the two styles with identical code.  I think we should keep the existing style in this file or move everything to `=>`